docs/source/concepts.rst

```diff
@@ -0,0 +1,16 @@
+Concepts in napari
+==================
+
+The Viewer
+----------
+
+At the heart of napari is the viewer model.  For a tutorial introduction to the
+viewer, see the `napari viewer tutorial <https://napari.org/tutorials/viewer>`_.
+
+The :class:`~napari.viewer.Viewer` object contains, among other things:
+
+- the main canvas
+- an instance of :class:`~napari.components.layerlist.LayerList` - a list of
+  layer objects that have been added to the :class:`~napari.viewer.Viewer`
+- an instance of
```

Comment:
Can we remove this for now. I'd like to merge what we have which does a lot of reorg stuff and then make dedicated follow up PRs for content. Would be nice not to have something trailing off ...... :-)

Reply:
cleaned it up.  Plugins.rst is still there, but I removed all references to it, so it's mostly undiscoverable.  I'll add to it soon, but we can merge this if you want.